docs/UserGuide.md

```diff
@@ -142,6 +142,19 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Adding a deadlines to meet in relation to a client : `deadline`
+
+Creates a deadline that is placed under the profile of a client.
+
+Format : `deadline INDEX DATE`
+
+- The given date is added to the client’s contact as deadline.
+- Date should be dd/mm/yyyy
+
+Example:
+
+- `deadline 1 01/01/2022` adds the date `01/01/2022` to the client contact in index `1`.
+
```

Comment:
Perhaps you could also include this feature/command in the **command summary** section at the bottom of the User Guide.

Reply:
Okay.